code/modules/mob/living/simple_animal/bot/construction.dm

```diff
@@ -166,7 +166,7 @@
 						return
 					newname = "redtag ED-209 assembly"
 				if("")
-					if(!istype(W, /obj/item/gun/energy/e_gun/advtaser))
+					if(!istype(W, /obj/item/gun/energy/e_gun/))
```

Comment:
I don't think these will ever be built if they require an out and out energy gun.

Reply:
When would sec be willing to give out a hybrid tazer but not an energy gun?  Especially when, given the crappy damage of the lethals mode, the tazer is almost certainly better?